frontend/ui/elements/font_ui_fallbacks.lua

```diff
@@ -0,0 +1,160 @@
+local FFIUtil = require("ffi/util")
+local Font = require("ui/font")
+local FontList = require("fontlist")
+local InfoMessage = require("ui/widget/infomessage")
+local UIManager = require("ui/uimanager")
+local util = require("util")
+local _ = require("gettext")
+local T = FFIUtil.template
+
+-- Some "Noto Sans *" fonts are already in ui/font.lua Font.fallbacks,
+-- because they are required to display supported UI languages.
+-- Don't allow them to be disabled.
+-- Duplicated from (and should be kept in sync with) ui/font.lua, but here
+-- by face name and there by font file/path name.
+local hardcoded_fallbacks = {
+    "Noto Sans CJK SC",
+    "Noto Sans Arabic UI",
+    "Noto Sans Devanagari UI",
+}
+-- Add any user font after Noto Sans CJK SC in the menu
+local additional_fallback_insert_indice = 2 -- (indice in the above list)
+
+local alt_name = {
+    -- Make it clear this "CJK SC" font actually supports all other CJK variant
+    ["Noto Sans CJK SC"] = "Noto Sans CJK SC (TC, JA, KO)"
+}
+
+local fallback_candidates = nil
+local fallback_candidates_path_to_name = nil
+
+local genFallbackCandidates = function()
+    if fallback_candidates then
+        return
+    end
+    fallback_candidates = {}
+    fallback_candidates_path_to_name = {}
+    for _, font_path in ipairs(FontList:getFontList()) do
+        local fontinfo = FontList.fontinfo[font_path] -- (NotoColorEmoji.tff happens to get no fontinfo)
+        if fontinfo and #fontinfo == 1 then -- Ignore font files with multiple faces
+            fontinfo = fontinfo[1]
+            if util.stringStartsWith(fontinfo.name, "Noto Sans ") and
+                        not fontinfo.bold and not fontinfo.italic and
+                        not fontinfo.serif and not fontinfo.mono then
+                fallback_candidates[fontinfo.name] = fontinfo
+                fallback_candidates_path_to_name[font_path] = fontinfo.name
+            end
+        end
+    end
+end
+
+local more_info_text = [[
+If some of your book titles, info, chapters…, or dictionary and wikipedia results, are not displayed well but shown as ￾￾ or ��, you may need to download the required fonts for their languages and enable them as additional UI fallback fonts.
```

Comment:
Perhaps some more brevity? I think our intelligent users will get the point. ;-D

```suggestion
If some book titles, dictionary entries and such are not displayed well but shown as ￾￾ or ��, it may be necessary to download the required fonts for those languages. They can then be enabld as additional UI fallback fonts.
```

Reply:
OK. Just for confirmation (and Grammatical Learnings of English for Make Benefit Glorious Nation of France):
Is "those" in "for those languages" correct ?
In singular: "If a book title is bad, it may be necessary to download the font for this language."
Which "this" language ? Is it correct and implicite ?
I would write "for its language", the title's language.

Reply:
It's not strictly entirely correct, but I'm not convinced a more correct phrase like "the relevant languages" reads any better. Perhaps "their associated languages" would do a better job of sounding both okay and being correct.